packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -121,7 +121,7 @@ ReferenceArrayField.propTypes = {
 
 export interface ReferenceArrayFieldProps
     extends PublicFieldProps,
-        InjectedFieldProps {
+        InjectedFieldProps<any> {
```

Comment:
Equivalent to what we had before. Will be addressed when working on the fields. Same for the others